debian.Dockerfile

```diff
@@ -0,0 +1,87 @@
+ARG PG_VERSION
+ARG PREV_TS_VERSION=1.6.1
+ARG PREV_EXTRA
+############################
+# Build tools binaries in separate image
+############################
+ARG GO_VERSION=1.14.2
+FROM golang:${GO_VERSION}-buster AS tools
+
+ENV TOOLS_VERSION 0.8.1
+
+RUN apt-get update && apt-get install -y git \
+    && rm -rf /var/lib/apt/lists/* \
+    && mkdir -p ${GOPATH}/src/github.com/timescale/ \
+    && cd ${GOPATH}/src/github.com/timescale/ \
+    && git clone https://github.com/timescale/timescaledb-tune.git \
+    && git clone https://github.com/timescale/timescaledb-parallel-copy.git \
+    # Build timescaledb-tune
+    && cd timescaledb-tune/cmd/timescaledb-tune \
+    && git fetch && git checkout --quiet $(git describe --abbrev=0) \
+    && go get -d -v \
+    && go build -o /go/bin/timescaledb-tune \
+    # Build timescaledb-parallel-copy
+    && cd ${GOPATH}/src/github.com/timescale/timescaledb-parallel-copy/cmd/timescaledb-parallel-copy \
+    && git fetch && git checkout --quiet $(git describe --abbrev=0) \
+    && go get -d -v \
+    && go build -o /go/bin/timescaledb-parallel-copy
+
+############################
+# Grab old versions from previous version
+############################
+ARG PG_VERSION
+FROM timescale/timescaledb:${PREV_TS_VERSION}-pg${PG_VERSION}${PREV_EXTRA} AS oldversions
```

Comment:
This line when I was testing was fetching from the Alpine image without the extras flag so it was like so:

```Dockerfile
FROM timescale/timescaledb:${PREV_TS_VERSION}-pg${PG_VERSION} AS oldversions
```